overlord/snapstate/backend_test.go

```diff
@@ -142,7 +142,7 @@ func (f *fakeStore) SnapInfo(spec store.SnapSpec, user *auth.UserState) (*snap.I
 	f.pokeStateLock()
 
 	sspec := snapSpec{
-		Name: spec.Name,
+		Name: snap.InstanceSnap(spec.Name),
```

Comment:
I was confused, here we don't need this, we need to check that Name doesn't have an instance-key, this code might also be unused right now though